javaparser-core/src/main/java/com/github/javaparser/ast/observer/ObservableProperty.java

```diff
@@ -50,6 +49,7 @@ public enum ObservableProperty {
     COMPONENT_TYPE(Type.SINGLE_REFERENCE),
     CONDITION(Type.SINGLE_REFERENCE),
     CONTENT(Type.SINGLE_ATTRIBUTE),
+    DEFAULT(Type.SINGLE_ATTRIBUTE),
```

Comment:
Why have you created this new entry?

Reply:
It was generated automatically

Reply:
This is something I'm discovering with your comments. Thank you.